classes/class-dintero-checkout-gateway.php

```diff
@@ -268,5 +268,18 @@ public function process_redirect_payment( $order ) {
 		public function process_refund( $order_id, $amount = null, $reason = '' ) {
 			return Dintero()->order_management->refund_order( $order_id, $reason );
 		}
+
+		/**
+		 * Check if the order can be refunded with Dintero or not.
+		 *
+		 * @param WC_Order $order The WooCommerce order.
+		 *
+		 * @return bool
+		 */
+		public function can_refund_order( $order ) {
+			$capture_date = $order->get_meta( '_wc_dintero_captured' );
+
+			return ! empty( $capture_date );
+		}
```

Comment:
If I understood this PR correctly, you're supposed to display a message if the order cannot be refunded. Where is this function used for that purpose?